pkg/spec/createconfig.go

```diff
@@ -444,6 +444,9 @@ func (c *CreateConfig) GetContainerCreateOptions(runtime *libpod.Runtime) ([]lib
 	if len(c.HostAdd) > 0 {
 		options = append(options, libpod.WithHosts(c.HostAdd))
 	}
+	if c.Hostname != "" {
```

Comment:
Shouldn't need this - I think we already set the hostname in the OCI spec when we create the container